app/views/school-search/partials/filter-mats.html

```diff
@@ -0,0 +1,68 @@
+<h2 class="govuk-heading-m">Filter MATs</h2>
+{{ govukButton({
+    text: "Apply filters",
+    name: "apply-mat-filters"
+}) }}
+
+            {{ govukInput({
```

Comment:
ooof

Is there an auto-formatter for nunjucks out there?

Reply:
Eesh, sorry :/
Not a good one that I could find. I'm using the Twig plugin for IntelliJ. VSCode does a slightly better job with it's Jinja plugin, but not much